pipelines/build/common/build_base_file.groovy

```diff
@@ -247,8 +247,12 @@ class Builder implements Serializable {
 
             }
         }
-
+        
         testList.unique()
+        
+        if (testList.contains('sanity.external') && configuration.arch != 'ppc64le' && configuration.arch != 's390x' && !(configuration.os == 'linux' && configuration.arch == 'x64')) {
+            testList.remove(testList.indexOf('sanity.external'))
+        }
```

Comment:
This is quite long conditional. Could we perhaps add a comment explaining why this is necessary with a link to the relevant issue?